### PR TITLE
Add configDB table names to schema.h

### PR DESCRIPTION
**Pull request by JipanYanga**
Signed-off-by: Jipan Yang <jipan.yang@alibaba-inc.com>

### CODE REVIEW

**Comment by lguohan** [comment]
@taoyl-ms  to review.

File: common/schema.h [hunk 1]
```diff
@@ -73,6 +73,16 @@ namespace swss {
 #define PLUGIN_TABLE "PLUGIN_TABLE"
 #define SAI_OBJECT_TYPE "SAI_OBJECT_TYPE"
 
+/***** CONFIGUATION DATABASE *****/
+#define CFG_PORT_TABLE_NAME             "PORT"
+#define CFG_INTF_TABLE_NAME             "INTERFACE"
+#define CFG_LAG_INTF_TABLE_NAME         "PORTCHANNEL_INTERFACE"
```
Comment (stcheng) [inline common/schema.h]: why we're using LAG in the TABLE name but PORTCHANNEL in the actual string?

Comment (JipanYanga) [reply common/schema.h]: Using LAG in the TABLE name macro to be consistent with app and state DB, it is less confusing when writing code.

Real table name PORTCHANNEL_INTERFACE is from minigraph, would like to keep it as it is at least for now.

Comment (stcheng) [reply common/schema.h]: ok

**Pull request merged**